### ISSUE TITLE
Can't create postgres engine

### ISSUE THREAD

**Issue by BulbaWarrior** [sqlalchemy/sqlalchemy#6105]
**Describe the bug**

Trying to create postgres database engine produces `sqlalchemy.exc.NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:postgres`

**To Reproduce**
```
docker run -it python:3 /bin/bash
pip install sqlalchemy
python
>>> from sqlalchemy import create_engine
>>> engine = create_engine('postgres://')
```

**Please do not use Flask-SQLAlchemy or any other third-party extensions or dependencies in test cases**.  The test case must illustrate the problem **without** using any third party SQLAlchemy extensions.  Otherwise, please report the bug to those projects first.

```py
import os
from sqlalchemy import create_engine
from sqlalchemy.orm import scoped_session, sessionmaker
from sqlalchemy.ext.declarative import declarative_base

user = os.environ['POSTGRES_USER']
pwd = os.environ['POSTGRES_PASSWORD']
db = os.environ['POSTGRES_DB']
host = 'db'
port = '5432'
engine = create_engine('postgres://%s:%s@%s:%s/%s' % (user, pwd, host, port, db))
```

**Error**

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/services/flaskapp/src/database.py", line 11, in <module>
    engine = create_engine('postgres://%s:%s@%s:%s/%s' % (user, pwd, host, port, db)) 
  File "<string>", line 2, in create_engine
  File "/usr/local/lib/python3.9/site-packages/sqlalchemy/util/deprecations.py", line 298, in warned
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/sqlalchemy/engine/create.py", line 522, in create_engine
    entrypoint = u._get_entrypoint()
  File "/usr/local/lib/python3.9/site-packages/sqlalchemy/engine/url.py", line 653, in _get_entrypoint
    cls = registry.load(name)
  File "/usr/local/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 341, in load
    raise exc.NoSuchModuleError(
sqlalchemy.exc.NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:postgres
```

**Versions.**
 - OS: python:3 docker image
 - Python:3.9.2
 - SQLAlchemy:latest
 - Database: postgres:9.6.5
 - DBAPI:

**Additional context**

**Have a nice day!**

**Comment by CaselIT** [comment]
Hi,

See https://github.com/sqlalchemy/sqlalchemy/issues/6083

tldr `postgres` was long deprecated, you should use `postgreslq`

**Comment by emilytozier** [comment]
Dear BulbaWarrior!
Can you managed the problem with sqlalchemy by editing postgre on postgresql? I have the same issue, and changin does not work!
